app/scripts/modules/core/cluster/filter/clusterFilter.service.js

```diff
@@ -82,6 +82,17 @@ module.exports = angular
         return serverGroup.vpcName.toLowerCase() === vpcName.toLowerCase();
       }
 
+      if (filter.includes('tag:')) {
+        let match = false;
+        let [, tag] = filter.split('tag:');
+        _.each(serverGroup.tags || {}, function (val, key) {
+          if (val.toLowerCase().contains(tag.toLowerCase())) {
```

Comment:
pretty close - just need to change `contains` to `includes` and you should be good to go